cinch/roles/jenkins_slave/tasks/check_swarm_upstart.yml

```diff
@@ -0,0 +1,14 @@
+- name: check if swarm connected to the master with upstart
```

Comment:
Based on my research it does not, unfortunately.  systemd is way more feature-rich than upstart.  upstart's log/status functionality leaves much to be desired, and if you run 'initctl status swarm' you'll get a zero exit code back even though the state is anything but *OK*.